application/cms/forms.py

```diff
@@ -159,7 +159,7 @@ class MeasurePageForm(FlaskForm):
     internal_reference = RDUStringField(
         label="Measure code (optional)", hint="This is for internal use by the Race Disparity Unit"
     )
-    publication_date = DateField(label="Publication date", format="%Y-%m-%d", validators=[Optional()])
+    published_at = DateField(label="Publication date", format="%Y-%m-%d", validators=[Optional()])
```

Comment:
The convention I've seen elsewhere is for `_at` fields to be datetimes, but to use `_on` for dates - ie `published_on`. Would that work?

Reply:
Ah that convention makes sense. But given it will mean re-naming three other fields, I _might_ leave this for now. We can rename all four to use the `_on` suffix later. It's not hard, but it feels like an extra lump of complexity on this ticket that I think I'd like to avoid. Good suggestion though.

Reply:
Yeah, it's not critical. And maybe they should all be timestamps anyway?

Reply:
Timestamps would be nice.